### ISSUE TITLE
ideas to think about...

### ISSUE THREAD

**Issue by Xenocide122** [cballou/Turntable.FM-Squared#9]
-Upvote delay input ( 4 seconds before i start to bop... 20 seconds..) using a slider, or custom text entry field

-Multiple sounds to choose form for the notifications (mario coin,  record scratch.. so on..)

-Some kind of tagging for songs

-button to push songs to the bottom of the que

:)

**Comment by cballou** [comment]
I like the slider idea. It's currently already set to start at a random interval to ensure that TT.FM doesn't pick up on any consistent patterns in voting.

**Comment by Xenocide122** [comment]
Nice! You could use both, have the slider to set the approximate upvote time, and you can have it randomly choose a time close to that...

**Comment by FlamingCow** [comment]
Indeed, one of my favorite features from TTtools is the bottom button... puts a song at the bottom of the queue.  Very helpful!  Sadly TTtools is not compatible with TTsquared.
